server/src/test/java/org/elasticsearch/repositories/blobstore/BlobStoreRepositoryTests.java

```diff
@@ -354,20 +359,29 @@ private BlobStoreRepository setupRepo() {
         }
         AcknowledgedResponse putRepositoryResponse = client.admin()
             .cluster()
-            .preparePutRepository(repositoryName)
+            .preparePutRepository(TEST_REPO_NAME)
             .setType(REPO_TYPE)
             .setSettings(repoSettings)
             .setVerify(false) // prevent eager reading of repo data
-            .get();
+            .get(TimeValue.timeValueSeconds(10));
         assertThat(putRepositoryResponse.isAcknowledged(), equalTo(true));
 
         final RepositoriesService repositoriesService = getInstanceFromNode(RepositoriesService.class);
-        final BlobStoreRepository repository = (BlobStoreRepository) repositoriesService.repository(repositoryName);
+        final BlobStoreRepository repository = (BlobStoreRepository) repositoriesService.repository(TEST_REPO_NAME);
         assertThat("getBlobContainer has to be lazy initialized", repository.getBlobContainer(), nullValue());
         assertEquals("Compress must be set to", compress, repository.isCompress());
         return repository;
     }
 
+    @After
+    public void removeRepo() {
+        try {
+            client().admin().cluster().prepareDeleteRepository(TEST_REPO_NAME).get(TimeValue.timeValueSeconds(10));
```

Comment:
Apparently the existing tests all worked even if the repo already existed, but that makes them run differently in CI from when run individually which is kinda surprising